toolbackup/src/main/java/io/dockstore/toolbackup/client/cli/Client.java

```diff
@@ -352,11 +352,11 @@ protected void setupClientEnvironment() {
 
         // pull out the variables from the config
         String token = config.getString("token", "");
-        String serverUrl = config.getString("server-url", "https://www.dockstore.org:8443");
+        String serverUrl = config.getString("server-url", "https://www.dockstore.org:443/api");
```

Comment:
Typo? `8448` turned into `443`

Reply:
This is legit, we switched ports a ways back https://github.com/dockstore/compose_setup/pull/161

Reply:
You could optionally drop the port, since 443 is the default for https.

Reply:
Added to https://ucsc-cgl.atlassian.net/browse/SEAB-5551